iwyu_ast_util.h

```diff
@@ -902,6 +902,14 @@ bool IsBaseToDerivedMemPtrConvertible(const clang::Type* base_mem_ptr_type,
                                       const clang::Type* derived_mem_ptr_type,
                                       clang::Sema&);
 
+// Checks whether an object of the type 'to' may be copy-initialized from
+// an object of the type 'from'. SourceLocation must be valid, otherwise clang
+// may crash on a class template specialization.
+bool IsConvertible(clang::QualType from,
+                   clang::QualType to,
+                   clang::SourceLocation,
```

Comment:
It would be nice to give this parameter a name for documentation purposes. It looks like `from_loc`, based on how it's used in the impl, but maybe there's a better name.

Reply:
Yeah, the problem may appear when `from` is an uninstantiated class template, but "conversion location" looks more honest with regard to what is passed from `VisitTypeTraitExpr`. It is quite sad that a location is needed for the function that should be location-agnostic...